Carbon Sabre changes, Baton blocking, Webbing changes.

TsunamiAnt opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

### Regarding Sabres:
- Carbon Fiber Sabres now no longer spawn in lockers.
- You can now purchase them in the security vendor.
- Added sabres to the parace clothes crates, made the crates an actual in-code thing.
- Removed all the hardmapped instances and replaced them with the coded ones.
- They now contain 4 sets of formal officer clothing, Including sabres with sheaths.
- Security belts now fit the sabre, and have sprites showing as such.
- Webbings of course cannot carry the sabre.
- To compensate, I took the opportunity to increase their slot count from 7 to 9.

## Why It's Good For The Game

No one ever steals the sabres. Security was never actually intended to use them.

In order to preclude the typical choice that we put players through of "Sabre vs Stun", Sabres now fit into normal belts.

As a fun side-tangent of that, Webbings can now hold more items, but not the sabre.

My hope is that this will reduce the amount of officers that feel they need to "choose" between sabres and stun weapons, and lessen the punishment of people just wanting drip.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>

<img width="1234" height="681" alt="image" src="https://github.com/user-attachments/assets/f811c03e-f85f-4eed-8e5b-b7cb11112858" />

(wip belt overlay sprite, the finished one looks better)
<img width="778" height="260" alt="image" src="https://github.com/user-attachments/assets/797d762a-6835-40f8-8e0c-29c802a6baec" />

<img width="261" height="242" alt="image" src="https://github.com/user-attachments/assets/17c597b7-158b-4f14-adfa-d8a425939164" />

<img width="173" height="142" alt="image" src="https://github.com/user-attachments/assets/786d4576-efd5-47dc-aa94-dadfe5fcc2d9" />

Webbing at max cap. Notably only small items in here, no swordy.
<img width="648" height="328" alt="image" src="https://github.com/user-attachments/assets/4cb187f2-fba5-46c1-8aa1-98211b5f4ac8" />

New sprite featuring better perspective and a tasteful fade.
<img width="207" height="190" alt="image" src="https://github.com/user-attachments/assets/24c380ff-cc0a-4210-a1af-95e3ce0b294e" />

</details>

## Changelog
:cl:
tweak: moved the security sabres to the parade crate  & sectech.
tweak: made security belts able to carry the sabre.
tweak: added some more slots to the webbing, now that It can't carry one of the ""mainline"" security weapons.
fix: fixes some small, very weirdly similar mapping oversights on multiple maps.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TsunamiAnt committed:
Adds carbon fiber sabre to security loadouts.

Adds the carbon fiber sabre to security belts, webbing, lockers, and vending machines to provide an alternative melee weapon option for security personnel.

Additionally:
-Improves the stun baton, allowing for a harder hit when inactive.
-Adds an empty sabre belt to wardrobe vending machines.
-Fixes the security webbing description.

TsunamiAnt committed:
Moves formal crates to security armories

Moves formal uniform crates from security vending machines to armories in various maps.

This change ensures that security personnel have easier access to their uniforms and weapons, streamlining their response to threats and improving overall security effectiveness.

Rukofamicom commented on code/game/objects/items/melee/misc.dm:
```diff
@@ -142,7 +141,11 @@
 	item_state = "sabre_fiber"
 	force = 15
 	armour_penetration = 25
+	belt_icon_state = "sabre"
```
Doesn't this override the belt icon to match the captain's appearance instead of inheriting the security one?

TsunamiAnt replied:
This is the icon state of the overlay that is applied to belts to show visual items, so, prolly not!

Rukofamicom replied:
okay I misremembered the variable names then.

Rukofamicom commented on code/game/objects/items/melee/misc.dm:
```diff
@@ -142,7 +141,11 @@
 	armour_penetration = 25
+	belt_icon_state = "sabre"
 	sharpness = SHARP //No dismembering for security sabre without direct intent
+	custom_premium_price = 200
```
```suggestion
	custom_premium_price = 100
```

This is too expensive

Rukofamicom commented on code/game/objects/items/stunbaton.dm:
```diff
@@ -1,19 +1,21 @@
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
 
-	force = 8 				//BRUTE when off
+	force = 15 				//BRUTE when off
```
This will be applied during harmbaton attacks when the requested fix for that goes through. 
This will mean batons can do 40 stamina and 15 brute damage in the same attack.

TsunamiAnt replied:
Would it be possible to lower stamina damage a bit in turn?

Besides iunno about stuff not on master!

Rukofamicom replied:
It might not be necessary to do anything, the lasting brute damage will only matter if the stamina damage is ineffective **and** the harmbaton fix passes as it currently is.

Rukofamicom reviewed (commented):
When blocking was overhauled, the baton was intentionally skipped over for blocking due to just how much damage it deals in each strike, the highest single hit damage available in the entire game with exception for items that only exist via adminbus.  

Stun batons (along with heretic blades and two-handed axes) are the only items without blocking capability that are still expected to win against someone who does have blocking, each for a different reason. Because of this expectation of victory, it is an insanely bad idea to give blocking to any of these, as it will make them incredibly unfair to face in a melee fight. 

This is a :key: from me until blocking is removed or nerfed on the stun baton. I left two other remarks but those are not blocking requests or part of the key - I just want to open them for discussion.

TsunamiAnt committed:
Update code/game/objects/items/melee/misc.dm

Co-authored-by: Rukofamicom <oozelingcomplaints@gmail.com>

TsunamiAnt committed:
Set block_power to 0 for stun baton

TsunamiAnt committed:
removes blocking from abductor and cattleprod batons

TsunamiAnt committed:
Moves sabre to belt instead of melee item

The carbon fiber sabre was being inappropriately spawned as a melee weapon in some vending machines, where it should be a belt upgrade.
This change moves the sabre from the general melee category to the belt category.

PowerfulBacon reviewed (changes requested):
Atomise this PR into its 3 parts (Parade crate, baton blocking, inventory changes)

TsunamiAnt committed:
hopefully reverts baton changes
